### PR TITLE
Add non-normative note for event-level reports deletion for unexpired destination limit

### PR DESCRIPTION
**Pull request by linnan-github**
***
<a href="https://pr-preview.s3.amazonaws.com/linnan-github/conversion-measurement-api/pull/1351.html" title="Last updated on Jul 1, 2024, 5:37 PM UTC (f40ddfd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/attribution-reporting-api/1351/bacfb84...linnan-github:f40ddfd.html" title="Last updated on Jul 1, 2024, 5:37 PM UTC (f40ddfd)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -2890,6 +2890,15 @@ To <dfn>delete sources for unexpired destination limit</dfn> given a [=set=] of
         and |report|'s [=event-level report/trigger time=] is greater than or equal to |now|:
         1. [=set/Append=] |report|'s [=event-level report/report ID=] to |deletedEventLevelReports|.
         1. [=set/Remove=] |report| from the [=event-level report cache=].
+
+    Note: The privacy concern on leaking browsing history of destinations
+    deactivated for unexpired destination limit from [=event-level reports=]
+    whose [=event-level report/trigger time=] is earlier than |now| is mitigated
+    by the presence of [=obtain a fake report|fake reports=]. [=Event-level reports=]
+    whose [=event-level report/trigger time=] is greater than or equal to |now|
+    must be deleted to avoid exposing whether an [=attribution source=] has
```
Comment (apasel422) [inline index.bs:2899]:
```suggestion
    must be deleted to avoid exposing whether an [=attribution source=] has a
```

File: index.bs [hunk 2]
```diff
@@ -2890,6 +2890,15 @@ To <dfn>delete sources for unexpired destination limit</dfn> given a [=set=] of
         and |report|'s [=event-level report/trigger time=] is greater than or equal to |now|:
         1. [=set/Append=] |report|'s [=event-level report/report ID=] to |deletedEventLevelReports|.
         1. [=set/Remove=] |report| from the [=event-level report cache=].
+
+    Note: The privacy concern on leaking browsing history of destinations
```
Comment (apasel422) [inline index.bs:2894]: IMO this sentence is a bit hard to parse, especially because it's unclear what "the privacy concern" actually is.

**Review by apasel422 — CHANGES_REQUESTED** [review]

File: index.bs (hunk 2 above)
Comment (linnan-github) [reply index.bs:2894]: The privacy concern is `leaking browsing history of destinations...`. Please let me know if you have better suggestion to rephrase. Thanks.

Comment (linnan-github) [reply index.bs]: Updated, thanks.

**Review by apasel422 — APPROVED** [review]
